README.md

```diff
@@ -43,10 +45,33 @@ Exec type options:<br />
 The "Settings" window on a mod displays the Command Line to play that mod directly via Quick Launch. You can copy that Quick Launch Command Line and use it to create a shortcut to easily launch that mod. <br />
 
 <br /><br />
-## **Software Rendering Mode:**<br />
-By default KnossosNET will render the UI in the GPU, can be set to run completely on software rendering what effectively avoids any use of the user GPU, this will come at the cost of increased CPU usage, what should not be a problem when in idle.
+## **Tool Quick Launch:**<br />
+Just like mods QuickLaunch, you can launch installed mod development tools with a cmdline argument. Windows users have the possibility to create a shortcut to desktop with a click. This may be extended to Linux and Mac later on.<br />
+KnossosNET.exe -tool {tool-id}
+
+<br /><br />
+## **Tray Icon Mode:**<br />
+On 1.3.0 a feature to "close to tray" has been implemented, this can be enabled on global settings.<br />
+Additionally, a cmdline argument (-traymode) has been added to Knossos.NET. By starting in this mode the launcher will start to tray and will not load UI resources until the user opens the UI for the first time.<br />
+The tray icon allow to launch mods directly from there.<br />
+![TrayIcon](https://i.imgur.com/cWF4WdT.png)
+
+<br /><br />
+## **Rendering Mode:**<br />
+From 1.3.0 KnossosNET will render the UI in software mode by default to avoid use of the user GPU, but it can be set to run on hardware gpu rendering.
+<br />
+You can force the hardware rendering mode by using the "-hardware" Command Line argument or by setting an environment variable "KNET_RENDER_MODE" to "hardware".<br />
+For versions older than 1.3.0 hardware rendering is used by default and can be set to run on software mode by using "-software" or KNET_RENDER_MODE" to "software".
+
+<br /><br />
+## **Mod Filters and tags:**<br />
+![tags](https://i.imgur.com/KKVyuxc.png)
 <br />
-You can force the software rendering mode by using the "-software" Command Line argument or by setting an environment variable "KNET_RENDER_MODE" to "software".
+A filter and tags feature offers QOL improvements to mod search and listening.<br />
+Filters are a physical button that will display all mods that contain it.<br />
```

Comment:
What do you think about the following change for clarity?
`Filters are a physical button that will display all mods that contain that filter.`